pkg/game/game.go

```diff
@@ -40,26 +40,32 @@ type Game struct {
 	canSwapTiles  bool
 }
 
-func NewFromTileSet(tileSet tilesets.TileSet, log logger.Logger) (*Game, error) {
+func NewFromTileSet(tileSet tilesets.TileSet, log logger.Logger, playerCount uint8) (*Game, error) {
 	deckStack := stack.New(tileSet.Tiles)
 	deck := deck.Deck{
 		Stack:        &deckStack,
 		StartingTile: tileSet.StartingTile,
 	}
-	return NewFromDeck(deck, log)
+	return NewFromDeck(deck, log, playerCount)
 }
 
 func NewFromDeck(
-	deck deck.Deck, log logger.Logger,
+	deck deck.Deck, log logger.Logger, playerCount uint8,
 ) (*Game, error) {
 	if log == nil {
 		nullLogger := logger.New(io.Discard)
 		log = &nullLogger
 	}
+
+	var players = []elements.Player{}
+	for i := range playerCount {
+		players = append(players, player.New(elements.ID(i+1)))
+	}
```

Comment:
Allocate a slice of right size from the beginning:
```suggestion
	var players = make([]elements.Player{}, playerCount)
	for i := range playerCount {
		players[i] = player.New(elements.ID(i+1))
	}
```

Reply:
changed